pkg/loadbalancer/experimental/redirectpolicy/table.go

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package redirectpolicy
+
+import (
+	"github.com/cilium/hive/job"
+	"github.com/cilium/statedb"
+	"github.com/cilium/statedb/index"
+	"k8s.io/client-go/tools/cache"
+
+	"github.com/cilium/cilium/pkg/k8s"
+	ciliumv2 "github.com/cilium/cilium/pkg/k8s/apis/cilium.io/v2"
+	"github.com/cilium/cilium/pkg/k8s/client"
+	k8sUtils "github.com/cilium/cilium/pkg/k8s/utils"
+	lb "github.com/cilium/cilium/pkg/loadbalancer"
+)
+
+const (
+	LRPTableName = "localredirectpolicies"
+)
+
+var (
+	lrpIDIndex = statedb.Index[*LocalRedirectPolicy, k8s.ServiceID]{
+		Name: "id",
+		FromObject: func(obj *LocalRedirectPolicy) index.KeySet {
+			// FIXME: Figure out which type to use for k8s object names. NamespacedName?
```

Comment:
Is this something that needs addressing now?

Reply:
Not really. More of a long term cleanup that we should do later. My thinking was we'd do perhaps a name type that is cluster+namespace+name and is just one string + indexes of the '/'s. This would avoid a lot of unnecessary string concatenations. But not strictly needed so I'll follow up with that much later.

Reply:
Removed the comment. I'll get to this later! :)